app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CartsController < ApplicationController
+  before_action :load_book, only: :create
+
+  def create
+    unless check_quantity
+      return respond_to do |format|
+        format.js
+      end
+    end
+    add_book_to_cart(params[:book_id], params[:quantity])
+    respond_to do |format|
+      format.js
+    end
```

Comment:
cơ mà code này lặp nhỉ

```suggestion
    add_book_to_cart(params[:book_id], params[:quantity]) if check_quantity
    respond_to do |format|
      format.js
    end
```